meson.build

```diff
@@ -1,6 +1,6 @@
 project('welcome',
-          version: '0.1.0',
-    meson_version: '>= 0.62.0',
+          version: '0.0.2',
+    meson_version: '>= 0.61.0',
```

Comment:
meson_version becomes smaller?

Reply:
yes 0.62 isnt available on non sid releases and i think ubuntu